src/main/resources/application.properties

```diff
@@ -10,9 +10,9 @@ spring.mail.properties.mail.smtp.ssl.trust: smtp.gmail.com
 spring.mail.properties.mail.smtp.connectiontimeout: 5000
 spring.mail.properties.mail.smtp.timeout: 5000
 spring.mail.properties.mail.smtp.writetimeout: 5000         
-spring.datasource.url=jdbc:mariadb://mariadb/welcome-bot
+spring.datasource.url=jdbc:mysql://localhost:3306/spring_social?serverTimezone=GMT-6
```

Comment:
Interesting choice of timezone. UTC is preferred way to go.

Reply:
Resolved.